docs/installation.md

```diff
@@ -0,0 +1,64 @@
+---
+hide:
+  - navigation
+---
+
+# Installation
+
+## Prerequisite
+
+To run Taipy, Python (version 3.8 or above) and [pip][] are required. If you don't have pip installed, this [Python installation guide][] can guide you through the process, or you can follow the [official installation page][] of pip.
+
+You can also install pip in a [conda][] environment by using the following command:
```

Comment:
[]?
as well as in the rest of this document.

Reply:
I mean I don't see the point of moving these Xrefs outside the text body.
This just makes is harder to maintain.

Reply:
agree, this also follows from the previous file. I will change it accordingly!